apps/trading/client-pages/settings/settings.tsx

```diff
@@ -0,0 +1,16 @@
+import { t } from '@vegaprotocol/i18n';
+import { TelemetryApproval } from '../../components/welcome-dialog/telemetry-approval';
+
+export const Settings = () => {
```

Comment:
chore (blocking): Can you add an e2e test to visit the page and use the checkbox